### PR TITLE
Added ModelExporter

### PR DESCRIPTION
**Pull request by Nelonn**

### CODE REVIEW

**Comment by AlexProgrammerDE** [comment]
Hi, I see you just added a separate git repositiry inside of this repository. Please PR new files into the existing repository instead of making a brand new repository and new mod.

**Comment by AlexProgrammerDE** [comment]
This code also only works with 1.21. You need to add a file for every version 1.7-1.21

**Comment by extremeheat** [comment]
Multi version implementations would be nice if you have the time to write it but it's not a requirement as it's not required by anything in PrismarineJS at the moment.

**Comment by Nelonn** [comment]
> Hi, I see you just added a separate git repositiry inside of this repository. Please PR new files into the existing repository instead of making a brand new repository and new mod.

I'm on macOS and cannot compile 1.12.2 version, so i added separate project that i can build.

```
A problem occurred configuring project ':1.12.2'.
> Failed to notify project evaluation listener.
   > Could not resolve all files for configuration ':1.12.2:minecraftLibraries'.
      > Could not find java-objc-bridge-1.0.0-natives-osx.jar (ca.weblite:java-objc-bridge:1.0.0).
        Searched in the following locations:
            https://repo.maven.apache.org/maven2/ca/weblite/java-objc-bridge/1.0.0/java-objc-bridge-1.0.0-natives-osx.jar
   > Could not resolve all files for configuration ':1.12.2:minecraftLibraries'.
      > Could not find java-objc-bridge-1.0.0-natives-osx.jar (ca.weblite:java-objc-bridge:1.0.0).
        Searched in the following locations:
            https://repo.maven.apache.org/maven2/ca/weblite/java-objc-bridge/1.0.0/java-objc-bridge-1.0.0-natives-osx.jar

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

**Comment by Nelonn** [comment]
> Multi version implementations would be nice if you have the time to write it but it's not a requirement as it's not required by anything in PrismarineJS at the moment.

Entities don't differ much from version to version and I don't have time to implement this on legacy versions.

**Comment by AlexProgrammerDE** [comment]
> I'm on macOS and cannot compile 1.12.2 version, so i added separate project that i can build.

This commit will fix it: https://github.com/PrismarineJS/minecraft-data-generator-server/pull/22/commits/5757961d96578f1b94b26d5d26083901cc013da7

> Also I'm using mojang mappings because we shouldn't be dependent of one technology like fabric (I'm talking about yarn mappings)

The team uses yarn because it is consistent from 1.7.-1.21. Being despondent on fabric is good here since it helps maintaining the project.

**Comment by AlexProgrammerDE** [comment]
> Entities don't differ much from version to version and I don't have time to implement this on legacy versions.

There are enough differences to warrant a implementation per version. But only for 1.21 should be sufficient as extremeheat said.

**Comment by AlexProgrammerDE** [comment]
Please add the code to the 1.21 module instead of adding a brand new submodule.

**Comment by Nelonn** [comment]
> > I'm on macOS and cannot compile 1.12.2 version, so i added separate project that i can build.
> 
> This commit will fix it: [5757961](https://github.com/PrismarineJS/minecraft-data-generator-server/commit/5757961d96578f1b94b26d5d26083901cc013da7)
> 
> > Also I'm using mojang mappings because we shouldn't be dependent of one technology like fabric (I'm talking about yarn mappings)
> 
> The team uses yarn because it is consistent from 1.7.-1.21. Being despondent on fabric is good here since it helps maintaining the project.

Merge your PR then I can work on this PR

**Comment by Nelonn** [comment]
Another issue on macOS after resetting entire gradle cache:

```
A problem occurred configuring project ':1.7'.
> Failed to notify project evaluation listener.
   > Could not resolve all files for configuration ':1.7:minecraftLibraries'.
      > Could not resolve org.lwjgl.lwjgl:lwjgl:2.9.1-nightly-20131017.
        Required by:
            project :1.7
         > Could not resolve org.lwjgl.lwjgl:lwjgl:2.9.1-nightly-20131017.
            > Could not parse POM https://libraries.minecraft.net/org/lwjgl/lwjgl/lwjgl/2.9.1-nightly-20131017/lwjgl-2.9.1-nightly-20131017.pom
               > Could not find org.lwjgl.lwjgl:parent:2.9.1-SNAPSHOT.
```

**Comment by rom1504** [comment]
Is this still useful? If yes then please finish it

File: 1.20.5/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 unimined.minecraft {
-    version "1.20.5"
```
Comment (AlexProgrammerDE) [inline 1.20.5/build.gradle:9]: Do not change the 1.20.5 module to 1.21. It needs to stay 1.20.5.

**Review by AlexProgrammerDE — CHANGES_REQUESTED** [review]

**Comment by rom1504** [comment]
Open another PR if you want this finished

**Pull request closed without merging**